addons/website_forum/controllers/main.py

```diff
@@ -550,6 +540,12 @@ def open_partner(self, forum, partner_id=0, **post):
 
     # Profile
     # -----------------------------------
+    def _get_validation_email_redirect_url(self, **kwargs):
```

Comment:
Instead of passing maybe 2 parameters that can be incoherent, why not give a unique param called `redirect_to` and its value can be 'slides' or 'forum' (aka, the path of the url /slides or /forum). So, no need to extend this method in other modules, and the url is simply 
`return '/' if not kwargs.get('redirec_to') else '/%s' % (kwargs['redirect_to'])`.
this will allow to redirect to any path: /forum/<id>, /slides, /, ...

Reply:
you are an amazing guy !